internal/utils/slice.go

```diff
@@ -0,0 +1,75 @@
+package utils
+
+import (
+	"fmt"
+	"ova-serial-api/internal/model"
+)
+
+var FORBIDDEN_VALUES = map[int]bool{-2: true, 1: true, 2: true, 5: true}
+
+func SplitSlice(slice []int, batchSize int) (newSlice [][]int) {
+	remainder := len(slice) % batchSize
+	newSlice = [][]int{}
+	for i := 0; i < len(slice); i += batchSize {
+		if i+batchSize <= len(slice) {
+			newSlice = append(newSlice, slice[i:i+batchSize])
+		}
+	}
+	if remainder > 0 {
+		newSlice = append(newSlice, slice[len(slice)-remainder:])
+	}
+	return
+}
+
+func SplitSerialSlice(serials []model.Serial, batchSize int) [][]model.Serial {
+	remainder := len(serials) % batchSize
+	var newSlice [][]model.Serial
+	for i := 0; i < len(serials); i += batchSize {
+		if i+batchSize <= len(serials) {
+			newSlice = append(newSlice, serials[i:i+batchSize])
+		}
+	}
+	if remainder > 0 {
+		newSlice = append(newSlice, serials[len(serials)-remainder:])
+	}
+	return newSlice
+}
+
+func FilterIntSlice(slice []int) (filtered []int) {
+	for i := range slice {
+		if !FORBIDDEN_VALUES[slice[i]] {
+			filtered = append(filtered, slice[i])
+		}
+	}
+	return
+}
+
+func Print2dIntSlice(slice [][]int) {
```

Comment:
Я думаю, что эти методы для принта не нужны. Их лучше убрать совсем. Хватит встроенного функционала Go для отображения результата. 
Если прямо ооочень хочется prettyPrint – можно либо попробовать использовать маршаллизацию (см. [здесь](https://siongui.github.io/2016/01/30/go-pretty-print-variable/), но надо держать в голове, что это очень дорогая операция и в реальном мире так делать нельзя), либо по крайней мере написать общий метод:
```suggestion
func Print2dSlice(slice [][]interface{}) {
```